src/components/plots/MultiViewGrid.jsx

```diff
@@ -88,7 +191,7 @@ const MultiViewGrid = (props) => {
               {
                 _.times(multiViewConfig.ncols, (j) => (
                   <Col flex key={multiViewConfig.ncols * i + j}>
-                    {plots[multiViewConfig.plotUuids[multiViewConfig.ncols * i + j]]}
+                    {plots[multiViewConfig.plotUuids[multiViewConfig.ncols * i + j]] || <div />}
```

Comment:
In the api we use `||` because there's no other option. In the UI please remember to use `??` instead wherever you want to add this.

By the way, what happens if the div is not added?

Reply:
|| does the job fine. Why do you prefer "??"

Reply:
I was wondering the same

Reply:
The sub configs for the mini plots might not be loaded yet so if the div is not there, it will crash